File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/ImageWidget.java
package com.pkmnapps.activitydo.dataclasses;

public class ImageWidget {
    String uid,aid;
    String imageUri;


    public ImageWidget(String uid, String aid, String imageUri) {
        this.uid = uid;
        this.aid = aid;
        this.imageUri = imageUri;
    }

    public String getImageUri() {
        return imageUri;
    }

    public void setImageUri(String imageUri) {
        this.imageUri = imageUri;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public String getAid() {
        return aid;
    }

    public void setAid(String aid) {
        this.aid = aid;
    }
}

File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/ImageWidget.kt
package com.pkmnapps.activitydo.dataclasses

import org.junit.runner.RunWith

class ImageWidget(var uid: String?, var aid: String?, var imageUri: String?) {
    fun getImageUri(): String? {
        return imageUri
    }

    fun setImageUri(imageUri: String?) {
        this.imageUri = imageUri
    }

    fun getUid(): String? {
        return uid
    }

    fun setUid(uid: String?) {
        this.uid = uid
    }

    fun getAid(): String? {
        return aid
    }

    fun setAid(aid: String?) {
        this.aid = aid
    }
}